### PR TITLE
Cvsl 2339 highlight hdc cases on approver caselist

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/views/pages/approve/cases.njk [hunk 1]
```diff
@@ -17,6 +17,9 @@
     {% elseif case.isDueForEarlyRelease and approvalNeededView %}
         {% set releaseDate = '<span class="urgent-highlight">' + case.releaseDate
             + '</span><span class="urgent-highlight urgent-highlight-message">Early release</span>' %}
+    {% elseif case.kind == 'HDC' and approvalNeededView %}
+        {% set releaseDate = '<span class="urgent-highlight">' + case.releaseDate
+            + '</span><span class="urgent-highlight urgent-highlight-message">HDC release</span>' %}
```
Comment (andrewrlee) [inline server/views/pages/approve/cases.njk:22]: mmh.. not sure if we need to do it for both approval needed and the approval given view

File: server/views/pages/approve/cases.test.ts [hunk 2]
```diff
@@ -32,4 +33,19 @@ describe('View and print a licence - case list', () => {
     expect($('#nomis-id-2').text()).toBe('A1234AB')
     expect($('#release-date-2').text()).toBe('1 Sep 2022')
   })
+
+  it('should display HDC release warning label when kind is HDC', () => {
```
Comment (andrewrlee) [inline server/views/pages/approve/cases.test.ts:37]: could add another test for approval needed : false to show it not shown

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/views/pages/approve/cases.njk (hunk 1 above)
Comment (louisenorris) [reply server/views/pages/approve/cases.njk:22]: Sorry I'm confused - assume `approvalNeededView` is the view we do want to show it in? and therefore that's what the above is doing? Is approval given view for `'/pdf-print'`? I think I need to keep `and approvalNeededView` 😅

Comment (louisenorris) [reply server/views/pages/approve/cases.njk]: Design confirmed it's needed for both approval needed and recently approved tab views so updated for this

File: server/views/pages/approve/cases.test.ts (hunk 2 above)
Comment (louisenorris) [reply server/views/pages/approve/cases.test.ts]: Added another test to cover highlighting HDC cases in both approval needed and recently approved tab views.

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**